<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestToggleButtonMainDialog</class>
 <widget class="QDialog" name="QtTestToggleButtonMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestToggleButton</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Games/GameBoenken/GameBoenken.qrc">
    <normaloff>:/images/R.png</normaloff>:/images/R.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ribi::QtToggleButtonWidget" name="toggle_button" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="label_is_pressed">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Is pressed: ?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_color">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Color: (?.?,?)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Change the toggle button its color:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDial" name="dial">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ribi::QtToggleButtonWidget</class>
   <extends>QWidget</extends>
   <header>qttogglebuttonwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Games/GameBoenken/GameBoenken.qrc"/>
 </resources>
 <connections/>
</ui>
